common/models/agent-trace.json

```diff
@@ -5,15 +5,18 @@
     "processId": {
       "type": "number",
       "description": "id of ServiceProcess, not the pid",
-      "required": true
+      "required": true,
+      "postgresql": {"column": "processId"}
```

Comment:
by default postgres would name this column `processid` and that messes up the mapping when it is read back into JS. Adding this column mapping fixes this.